ruby/src/api.cc

```diff
@@ -474,7 +469,7 @@ namespace leatherman { namespace ruby {
                                                                   "break file if File.exist? file;"
                                                                   "false end)" });
         if (!success) {
-            throw execution_failure_exception("ruby failed to run", output, none);
+            LOG_WARNING("ruby failed to run", output, none);
```

Comment:
This should raise a format exception from boost.  There's no `%1%` or `%2%` in the message.